### ISSUE TITLE
Tests fail with wrapt<1.10

### ISSUE THREAD

**Issue by douardda** [tantale/deprecated#9]
Current setup.py express the dependency on wrapt as:
```
 install_requires=[
        'wrapt < 2, >= 1',
    ],
```

However using wrapt 1.9.0 (since it's the current version available on debian stretch) fails with:

```
____________________ ERROR collecting tests/test.py _____________________
tests/test.py:4: in <module>
    import deprecated
deprecated/__init__.py:13: in <module>
    from deprecated.classic import deprecated
deprecated/classic.py:18: in <module>
    class ClassicAdapter(wrapt.AdapterFactory):
E   AttributeError: module 'wrapt' has no attribute 'AdapterFactory'
```
(as well as in other tests).

Using wrapt 1.10+ works as expected.

* Python version: 3.5
* Deprecated version: 3dc742c571de7cebbbdaaf4c554f2f36fc61b3db

**Comment by tantale** [comment]
Deprecated was tested with wrapt v1.11.1.

The latest minor releases are: v1.0.0, v1.1.4, v1.2.1, v1.3.1, v1.4.2, v1.5.1, v1.6.0, v1.7.0, v1.8.0, v1.9.0, v1.10.11, v1.11.2. 

I'm investigating this issue… Stay tuned.

**Comment by tantale** [comment]
Currently, Deprecated uses `wrapt.AdapterFactory` which is available since wrapt v1.10 or above.

I do not intend to modify the source code to support an old version of Wrapt.

But, I can change the constraint to the requirements:

```python
 install_requires=[
        'wrapt < 2, >= 1.10',
    ],
```

**Comment by tantale** [comment]
To solve your problem, you can add this constraint in your own `setup.py`:

```python
 install_requires=[
        'deprecated',
        'wrapt < 2, >= 1.10',
        ...
    ],
```

I hope this will suit you needs.
